templates/aframe-model-template/app/assets/index.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Default scene! • Made with A-Frame</title>
+    <link rel="stylesheet" href="css/app.css">
+    <script src="js/vendor.js"></script>
+    <script src="js/app.js"></script>
+    <script>
+    AFRAME.registerComponent('teleport', {
+      init: function () {
+        this.el.addEventListener('click', this.handleClick.bind(this));
+      },
+      handleClick: function (e) {
+        var camera = this.el.sceneEl.camera.el;
+        var hotspotPosition = e.target.getAttribute('position');
+        var hotspotRotation = e.target.getAttribute('rotation');
+        var cameraPosition = camera.getAttribute('position');
+        camera.setAttribute('position', {
+          x: hotspotPosition.x,
+          y: cameraPosition.y,
+          z: hotspotPosition.z
+        });
+        camera.setAttribute('rotation', {
+          y: hotspotRotation.y
+        });
+      }
+    });
+    </script>
+  </head>
+  <body>
+    <a-scene>
+      <a-assets>
+        <a-mixin id="hotspot" geometry="primitive: circle; radius: 0.20" material="color: lightblue"></a-mixin>
+        <a-mixin id="hotspot-select" geometry="primitive: cylinder; radius: 0.3; height: 2; openEnded: true" material="color: lightblue; side: double; opacity: 0.2"></a-mixin>
+        <a-mixin id="hotspot-arrow" rotation="0 0 0" geometry="primitive: cone; radiusTop: 0; radiusBottom: 0.05; height: 0.15;" material="color: rgb(87, 153, 255)"></a-mixin>
+      </a-assets>
+
+      <a-entity teleport position="0 0.1 0" rotation="-90 0 0" mixin="hotspot">
+        <a-entity mixin="hotspot-arrow"></a-entity>
+      </a-entity>
+      <a-entity teleport position="2 0.1 -2.5" rotation="-90 90 0" mixin="hotspot">
+        <a-entity mixin="hotspot-arrow"></a-entity>
+      </a-entity>
+      <a-entity teleport position="-2 0.1 -2.5" rotation="-90 -90 0" mixin="hotspot">
+        <a-entity mixin="hotspot-arrow"></a-entity>
+      </a-entity>
+      <a-entity teleport position="0 0.1 -5" rotation="-90 180 0" mixin="hotspot">
+        <a-entity mixin="hotspot-arrow"></a-entity>
+      </a-entity>
+
+      <a-entity position="0 0 -2.5" rotation="0 180 0">
+        <a-entity gltf-model="src: url(model/rocks.gltf)" scale="0.5 0.5 0.5"></a-entity>
```

Comment:
do you want to call this directory `models` (even though I know there's only one model)?